CareKit/SymptomTracker/OCKSymptomTrackerViewController.m

```diff
@@ -84,8 +84,7 @@ - (void)viewDidLoad {
                                                                               style:UIBarButtonItemStylePlain
                                                                              target:self
                                                                              action:@selector(showToday:)];
-    self.navigationItem.rightBarButtonItem.tintColor = self.progressRingTintColor;
-    
```

Comment:
@matteocrippa When I run the sample app using your branch, I get the following:

![simulator screen shot may 4 2016 12 21 36 pm](https://cloud.githubusercontent.com/assets/13040996/15026399/1cadf9ae-11f3-11e6-8602-643cd1a5fcdd.png)

Then I change the `progressRingTintColor` to purple, and I get the following:

![simulator screen shot may 4 2016 12 23 45 pm](https://cloud.githubusercontent.com/assets/13040996/15026412/2d2edcc6-11f3-11e6-8a5c-44f3bc9688c2.png)

Are you using the latest master? If so, could provide exact steps to reproduce the condition shown in your screenshot. Thanks! :)